app/views/internal/configs/show.html.erb

```diff
@@ -60,6 +60,20 @@
           </div>
         </div>
 
+        <div class="card mt-3">
+          <div class="card-header">Authentication</div>
+          <div class="card-body">
+            <div class="form-group">
+              <%= f.label :authentication_providers, "Authentication providers" %>
+              <%= f.text_field :authentication_providers,
```

Comment:
I worry a little about typos with this being a text field but since these are so restricted I can deal with it for now.

Reply:
Great catch, this can be a select